extensions.html

```diff
@@ -50,7 +50,7 @@
                 company: "Google", companyURL: "http://www.google.com/",
                 w3cid: 87824 },
           ],
-
+          xref: true,
```

Comment:
Please remove this `xref` in favor of the `xref` below with a list of specification short names.

Reply:
Done